README.md

```diff
@@ -1,28 +1,46 @@
-# FUN-MOOC
+# Richie site factory
 
-This repository is a work-in-progress for the future fun-mooc.fr website. It is
-based on [Richie](https://github.com/openfun/richie), a CMS for Open Education.
+This repository is a workbench for France Université Numérique to develop
+themed sites based on [Richie](https://github.com/openfun/richie), the CMS
+for Open Education.
 
-## Getting started
+## Prerequisite
 
 First, make sure you have a recent version of Docker and [Docker
 Compose](https://docs.docker.com/compose/install) installed on your laptop:
 
 ```
 $ docker -v
-  Docker version 1.13.1, build 092cba3
+  Docker version 19.03.10, build 9424aeaee9
 
 $ docker-compose --version
-  docker-compose version 1.17.1, build 6d101fb
+  docker-compose version 1.25.5, build 8a1c60f6
 ```
 
-The easiest way to start working on the project is to use our `Makefile`:
+## Getting started
+
+First, you need to activate the site on which you want to work. We provide
+a script that will list existing sites and let you choose one. Source it:
 
+```bash
+$ . bin/activate
```

Comment:
This won't work if user's default shell is not BASH. I would recommend to execute it instead:

```suggestion
$ bin/activate
```